website/content/docs/plugins/plugin-development.mdx

```diff
@@ -66,6 +75,29 @@ func main() {
 And that's basically it! You would just need to change `myPlugin` to your actual
 plugin.
 
+## Plugin Backwards Compatibility with Vault
+
+Let's take a closer look at a snippet from the above main package.
+
+```go
+	err := plugin.ServeMultiplex(&plugin.ServeOpts{
+		BackendFactoryFunc: myPlugin.Factory,
+        TLSProviderFunc:    tlsProviderFunc,
+	})
+```
+
+The call to `plugin.ServeMultiplex` ensures that the plugin will use
+Vault's plugin [plugin
```

Comment:
Suggest to remove additional "plugin" before [plugin multiplexing](/docs/plugins/plugin-architecture#plugin-multiplexing) link.